pages/index.vue

```diff
@@ -132,12 +144,41 @@ export default defineComponent({
   },
   methods: {
     async getTasks() {
+      this.loading = true;
       /***
        * @todo Fetch the tasks created by the user and display them.
        * @todo also the function to display a single new task added
        * @hints use store and set loading true
        * @caution you have to assign new value to todos for it to update
        */
+      let token = this.$store.getters.token;
+      //console.log(token)
+      this.todos = [];
```

Comment:
you could have let this be in prev state and only when you get response then change the array

Reply:
redid it that way